### PR TITLE
interface_attach_resource

### PR DESCRIPTION
**Pull request by volosatik**
closes CMPT-33768

### CODE REVIEW

File: templates/compute/resources/vkcs_compute_interface_attach.md.tmpl [hunk 1]
```diff
@@ -12,22 +12,10 @@ description: |-
...
-{{tffile "templates/compute/resources/vkcs_compute_interface_attach/fixed_ip/main.tf"}}
+{{tffile .ExampleFile}}
 
 ### Attachment Using an Existing Port
-{{tffile "templates/compute/resources/vkcs_compute_interface_attach/existing_port/main.tf"}}
-
-### Attaching Multiple Interfaces
-{{tffile "templates/compute/resources/vkcs_compute_interface_attach/multiple_interfaces/main.tf"}}
-
-Note that the above example will not guarantee that the ports are attached in a deterministic manner. The ports will be attached in a seemingly random order.
```
Comment (ftersin) [inline templates/compute/resources/vkcs_compute_interface_attach.md.tmpl:26]: why do not leave this?

File: examples/compute/interface_attach/existing-port/main.tf [hunk 2]
```diff
@@ -0,0 +1,4 @@
+resource "vkcs_compute_interface_attach" "etcd" {
```
Comment (ftersin) [inline examples/compute/interface_attach/existing-port/main.tf:1]: what is a reason for this example?

File: templates/compute/resources/vkcs_compute_interface_attach.md.tmpl [hunk 3]
```diff
@@ -12,22 +12,16 @@ description: |-
...
+{{tffile "examples/compute/interface_attach/main-port.tf"}}
 
-If you want to ensure that the ports are attached in a given order, create explicit dependencies between the ports, such as:
-{{tffile "templates/compute/resources/vkcs_compute_interface_attach/multiple_interfaces_ordered/main.tf"}}
-{{ .SchemaMarkdown }}
+Attaching multiple interfaces will not guarantee that interfaces are attached in
+a deterministic manner. The interfaces will be attached in a seemingly random
+order.
+If you want to ensure that interfaces are attached in a given order, create
+explicit dependencies between the interfaces , such as in examples/compute/volume_attach folder
```
Comment (ftersin) [inline templates/compute/resources/vkcs_compute_interface_attach.md.tmpl:24]: mention title of the example in doc not file path

File: templates/compute/resources/vkcs_compute_interface_attach.md.tmpl [hunk 4]
```diff
@@ -12,22 +12,16 @@ description: |-
 
 ## Example Usage
 ### Basic Attachment
```
Comment (ftersin) [inline templates/compute/resources/vkcs_compute_interface_attach.md.tmpl:14]: this is not a base case indeed. lets rename it

Comment (volosatik) [reply templates/compute/resources/vkcs_compute_interface_attach.md.tmpl:14]: renamed

**Review by ftersin — APPROVED** [review]

**Pull request merged**